### PR TITLE
warm up by adding a fake header div

### PR DESCRIPTION
**Pull request by calebx**
step by step

### CODE REVIEW

**Comment by Darmody** [comment]
LGTM

File: src/components/Header/index.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import _ from 'ramda'
+import setDisplayName from 'recompose/setDisplayName'
+import CSSModules from 'react-css-modules'
+import styles from './styles.scss'
+import Icon from 'components/Icon'
+
+const Header = () => (
+  <div styleName="nav-bar">
+    <Icon name="track_changes"/>
```
Comment (Darmody) [inline src/components/Header/index.js]: `track-changes`

File: src/components/Content/styles.scss [hunk 2]
```diff
@@ -1,12 +1,14 @@
 @import '~sass-flex-mixin/flex';
 @import '../../styles/foundation-theme';
 
+$gray_bg: #F1F1F1;
```
Comment (Darmody) [inline src/components/Content/styles.scss]: `$gray-bg`

我们是不是应该尽量用 `Foundation` 的颜色？

File: src/components/Header/index.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import _ from 'ramda'
+import setDisplayName from 'recompose/setDisplayName'
+import CSSModules from 'react-css-modules'
+import styles from './styles.scss'
+
+const Header = () => (
+  <div className="top-bar" styleName="top-bar">
```
Comment (Darmody) [inline src/components/Header/index.js]: Maybe we should create a topbar inside header?

File: src/components/VideoItem/index.js [hunk 4]
```diff
@@ -17,9 +17,9 @@ const VideoItem = ({ video }) => (
       </div>
       <div className="media-object-section" styleName="detail-section">
         <a href="#">
-          <h6 styleName="title" >{video.title}</h6>
-          <p styleName="description">{video.description}</p>
+          <h5 styleName="title" >{video.title}</h5>
         </a>
+        <p styleName="description">{video.description}</p>
```
Comment (Darmody) [inline src/components/VideoItem/index.js]: why move out?

File: src/styles/_settings.scss [hunk 5]
```diff
@@ -0,0 +1,576 @@
...
+// $tooltip-font-size: $small-font-size;
+// $tooltip-pip-width: 0.75rem;
+// $tooltip-pip-height: $tooltip-pip-width * 0.866;
+// $tooltip-radius: $global-radius;
+
+// 36. Top Bar
+// -----------
+
+// $topbar-padding: 0.5rem;
+$topbar-background: $white; // $light-gray;
```
Comment (Darmody) [inline src/styles/_settings.scss]: 为什么要把这种变量定义在这里呢，这像是组件内部的样式

File: src/styles/foundation.scss [hunk 6]
```diff
@@ -1,3 +1,3 @@
-@import 'foundation-settings';
+@import '_settings.scss';
```
Comment (Darmody) [inline src/styles/foundation.scss]:
```css
@import 'settings'
``` 
Is it work?

File: src/components/VideoItem/styles.scss [hunk 7]
```diff
@@ -1,31 +1,32 @@
...
+  box-shadow: none;
 }
 
 .title {
-  font-size: 16px;
-  font-weight: 500;
+  font-size: 1rem;
+  font-weight: 600;
   color: $primary-color;
+  font-family: $header-font-family;
```
Comment (Darmody) [inline src/components/VideoItem/styles.scss]: Why `header` here?

File: src/components/Header/index.js (hunk 3 above)
Comment (calebx) [inline src/components/Header/index.js]: good idea

File: src/components/VideoItem/index.js (hunk 4 above)
Comment (calebx) [inline src/components/VideoItem/index.js]: description usually can not be clickable.

File: src/styles/foundation.scss (hunk 6 above)
Comment (calebx) [inline src/styles/foundation.scss]: works

**Comment by Darmody** [comment]
Seems two commits can be squash into one.

File: src/components/Header/index.js (hunk 1 above)
Comment (Darmody) [inline src/components/Header/index.js]: it come back! 💀

File: src/components/Content/styles.scss (hunk 2 above)
Comment (Darmody) [inline src/components/Content/styles.scss]: ⬆️ 💀

File: src/styles/_settings.scss (hunk 5 above)
Comment (calebx) [inline src/styles/_settings.scss]: let _settings.scss control all needed variables.

File: src/components/VideoItem/styles.scss (hunk 7 above)
Comment (calebx) [inline src/components/VideoItem/styles.scss]: h1, h2, h3, this kind of header

**Comment by calebx** [comment]
ready to go?

**Comment by Darmody** [comment]
LGTM

**Pull request merged**